crypto/evp/evp_local.h

```diff
@@ -339,6 +341,23 @@ int evp_do_md_ctx_setparams(const EVP_MD *md, void *provctx,
 
 OSSL_PARAM *evp_pkey_to_param(EVP_PKEY *pkey, size_t *sz);
 
+/* copy from include/openssl/prov_ssl.h */
+#define SSL_MAX_MASTER_KEY_LENGTH 48
+
+#define SSL_MASTER_KEY(ctx, arglen) ({                                       \
```

Comment:
({ ... }) is a GNU extension and, as such, cannot be part of the source code.

Reply:
Thanks for the pointer. Addressed in the update